src/theme/NavbarItem/DefaultNavbarItem.js

```diff
@@ -0,0 +1,230 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Wrong copyright?

Reply:
I swizzled the original component and only modified the part between the Playwright start/end comments. Was by intention to not adjust it to have an easier diff in the future. Or is it necessary?